src/main/java/seedu/address/model/commission/Commission.java

```diff
@@ -139,4 +140,41 @@ public String toString() {
         return builder.toString();
     }
 
+    /**
+     * Builder class for Commission.
+     */
+    public static class CommissionBuilder {
```

Comment:
Shall we add the null checks back for CommissionBuilder? (yes i realise i also forgot)

so something like this https://github.com/AY2223S1-CS2103T-W11-3/tp/pull/79